debian/context/install.sh

```diff
@@ -159,7 +159,7 @@ then
 
         EFI_DISKS=$(blkid | grep "PARTLABEL=\"efi\"" | awk -F':' '{ print $1 }')
         for EFI_DISK in $EFI_DISKS; do
-            efibootmgr -c -d "${EFI_DISK}" -p1 -l \\EFI\\centos\\shimx64.efi -L "${BOOTLOADER_ID}"
+            efibootmgr -c -d "${EFI_DISK}" -p1 -l "\\EFI\\${BOOTLOADER_ID}\\grubx64.efi" -L "${BOOTLOADER_ID}"
```

Comment:
Shouldn't this just be `OS_NAME`?

Reply:
No, had a look into already installed machines and the efi directories are named as BOOLOADER_ID, eg. metal-debian and metal-ubuntu, also the name of the chainloader is grubx64.efi instead of shimx64.efi